yury-ortuno/lesson-1/homework-1-contacs-manager/src/Contact.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Created by Yury Ortuno on 11/13/2018.
+ */
+public class Contact {
+    private String name;
+    private String phoneNumber;
+
+    void setName(String name) {
+        this.name = name;
+    }
+
+    String getPhoneNumber() {
```

Comment:
You should use an identifier access for this method (public or private)

Reply:
Fix Done